content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -93,6 +96,42 @@ Pods. This means that an ill-behaved Pod that floods the API server with
 requests cannot prevent leader election or actions by the built-in controllers
 from succeeding.
 
+### Request Width
+
+The above description of concurrency management is the baseline story.
+In it, all requests have equal "width": each takes up one _unit of
+concurrency_, which we call a "seat" because this is similar to how
```

Comment:
Would “unit of effort” be appropriate?

PS. try to [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we)

Reply:
"effort" is too vauge.  For example, it sounds to me a lot like "work" --- which is the product of duration and width.

Reworded to not use "we".